src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -249,20 +249,19 @@ public synchronized void applyDDL(Sequenced<DDLEvent> sequencedEvent) throws Exc
         TableId tableId = TableId.of(project, event.getDatabase(), event.getTable());
         Table table = bigQuery.getTable(tableId);
         // TODO: check schema of table if it exists already
-        // TODO: add a way to get PK of a table through Delta API?
         if (table == null) {
           TableDefinition tableDefinition = StandardTableDefinition.newBuilder()
             .setSchema(Schemas.convert(addSequenceNumber(event.getSchema())))
             .build();
-          String description;
+          String primaryKey;
           if (event.getPrimaryKey().isEmpty()) {
-            description = "Primary Key: _sequence_num";
+            primaryKey = "_sequence_num";
           } else {
-            description = "Primary Key: " + event.getPrimaryKey().stream().collect(Collectors.joining(","));
+            primaryKey = event.getPrimaryKey().stream().collect(Collectors.joining(","));
```

Comment:
I believe this is the case, but BQ column names can't have commas right?

Reply:
Yes. BQ column name should not contain comma. I am using the comma here to join multiple primary keys if there is any. When I am get the string of primary key, we will split them by comma. From my testing, it works.

Reply:
hm actually the primary key here comes from the source. None of our current sources have commas, but there is no guarantee for the future.

I think it's more future-proof to use gson to serialize an entire object, even if it currently only contains a single `List<String>` for the primaryKey. This will let us add new information into the state in future versions if needed and it will still be backwards compatible.

Reply:
Sure. Agree, I will do that.